app/mailers/notification_mailer.rb

```diff
@@ -12,6 +12,13 @@ def notify_user_registration(user)
     mail(to: @user.email, subject: 'Welcome to Punchclock')
   end
 
+  def resend_user_registration(user)
+    @user = user
+    user.touch(:confirmed_at)
+    @token = user.send(:set_reset_password_token)
+    mail(to: @user. email, subject: 'Welcome to Punchclock', template_name: 'notify_user_registration')
```

Comment:
thoughts on using the same template from `notify_user_registration`? should I add a new one even though they're identical? (so this method is not dependent on how `notify_user_registration` looks).